packages/q/qt6gui/xmake.lua

```diff
@@ -7,7 +7,7 @@ package("qt6gui")
         package:data_set("libname", "Gui")
 
         if package:is_plat("linux") then 
-            package:add("deps", "freetype", "fontconfig", "libxkbcommon")
+            package:add("deps", "freetype", "fontconfig", "libxkbcommon", "libgl1-mesa-dev")
```

Comment:
this is not a xmake-repo package, it won't work

Reply:
Already pushed fixed commit.